Scripts/NodeDataCache.cs

```diff
@@ -60,6 +68,31 @@ public static class NodeDataCache {
                     ports.Add(staticPort.fieldName, port);
                 }
             }
+            
+            // Finally, make sure dynamic port settings correspond to the settings of their backing field
+            foreach (NodePort dynamicPort in dynamicPorts) {
+                string[] parts = dynamicPort.fieldName.Split(' ');
+                if (parts.Length != 2) {
+                    Debug.LogError("Dynamic port name " + dynamicPort.fieldName + " is invalid.");
+                    continue;
+                }
```

Comment:
Not an error. Not all dynamic ports are used by DynamicPortList

Reply:
Also bear in mind that a port can technically be named "myField 2" on a node with a valid `myField` variable *without* `myField` being a DynamicPortList. So you have to verify that `myField` is in fact a DynamicPortList before making any changes to the port.

Reply:
Proper verification for dynamic list ports is now in place, though I'd appreciate having it reviewed.